README.md

```diff
@@ -425,6 +426,26 @@ suc:
   version: v0.10.0
 ```
 
+### ArgoCD
+
+The ArgoCD bundle deploys [ArgoCD](https://argo-cd.readthedocs.io/en/stable/).
+
+To configure the bundle, use the `argocd` block:
+
+```yaml
+#cloud-config
+
+# Specify the bundle to use
+bundles:
+  - targets:
+      - run://quay.io/kairos/community-bundles:argocd_latest
+
+# Specify argocd settings
+argocd:
+  values: ....
```

Comment:
I'm a bit detached from the bundles so not sure if what I'm asking is not present in the other bundles, if that's the case feel free to ignore.

How about adding some example values here?

Reply:
I leaned on https://github.com/kairos-io/community-bundles/blob/main/README.md?plain=1#L194.
On other side, some other bundle have example: https://github.com/kairos-io/community-bundles/blob/main/README.md?plain=1#L236-L239